tutorials/cloud_access/euclid-cloud-access.md

```diff
@@ -0,0 +1,303 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.16.3
+kernelspec:
+  display_name: irsa-tutorials
+  language: python
+  name: python3
+---
+
+# Euclid Quick Release 1: Cloud Access
+
++++
+
+## Learning Goals
+- Learn where Euclid QR1 data is present in the cloud
+- Find an image and retireve its cutout from cloud
+- Find an object and retrieve its spectrum from cloud
+
++++
+
+## Introduction
+TODO: fill with links of docs/specs?
+
++++
+
+## Imports
+
+```{code-cell} ipython3
+# Uncomment the next line to install dependencies if needed.
+# !pip install s3fs astropy astroquery matploltib
+```
+
+```{code-cell} ipython3
+import s3fs
+from astropy.coordinates import SkyCoord
+import astropy.units as u
+from astropy.visualization import ImageNormalize, PercentileInterval, AsinhStretch
+from astropy.io import fits
+from astropy.nddata import Cutout2D
+from astropy.wcs import WCS
+from astropy.table import Table
+from astroquery.ipac.irsa import Irsa
+from matplotlib import pyplot as plt
+import json
+```
+
+## Browse Euclid QR1 Bucket
+
+```{code-cell} ipython3
+BUCKET_NAME='nasa-irsa-euclid-q1' # internal to IPAC until public release (use LAN or VPN w/ Tunnel-all)
+```
+
+```{code-cell} ipython3
+s3 = s3fs.S3FileSystem(anon=True)
+```
+
+```{code-cell} ipython3
+s3.ls(f'{BUCKET_NAME}/q1')
+```
+
+## Find images for a coordinate search
+
++++
+
+### Locate MER images in the bucket
+
+```{code-cell} ipython3
+s3.ls(f'{BUCKET_NAME}/q1/MER')
+```
+
+```{code-cell} ipython3
+s3.ls(f'{BUCKET_NAME}/q1/MER/102018211')
+```
+
+```{code-cell} ipython3
+s3.ls(f'{BUCKET_NAME}/q1/MER/102018211/VIS')
+```
+
+As per doc specification, we need `MER/{tile_id}/{instrument}/EUC_MER_BGSUB-MOSAIC*.fits` for displaying background-subtracted mosiac images. But these images are stored under TILE IDs so first we need to find TILE ID for a coordinate search we are interested in. We will use astroquery (in next section) to retrieve FITS file paths for our coordinates.
+
++++
+
+### Get image file paths for a coordinate search of interest
+
+```{code-cell} ipython3
+# Star TYC 4429-1677-1
+ra = 273.11730230 * u.deg
+dec = 68.20761349 * u.deg
+```
+
+```{code-cell} ipython3
+coord = SkyCoord(ra=ra, dec=dec)
+search_radius = 10 * u.arcsec
+```
+
+List all Simple Image Access (SIA) collections for IRSA.
+
+```{code-cell} ipython3
+tbl = Irsa.list_collections(servicetype='SIA')
+len(tbl)
+```
+
+Filter to only those containing "euclid":
+
+```{code-cell} ipython3
+tbl[['euclid' in v for v in tbl['collection']]]
+```
+
+We identify the collection we need for MER images:
+
+```{code-cell} ipython3
+img_collection = 'euclid_DpdMerBksMosaic'
+```
+
+```{code-cell} ipython3
+img_tbl = Irsa.query_sia(pos=(coord, search_radius), collection=img_collection).to_table()
```

Comment:
This is not yet a real suggestion as this is not yet released, but I change astroquery that `query_sia` will return an astropy QTable, so there will be no need for the `to_table()`

```suggestion
img_tbl = Irsa.query_sia(pos=(coord, search_radius), collection=img_collection)
```